### PR TITLE
MAT-8231 Generating ELM without any annotation filtered, Also using E…

### PR DESCRIPTION
**Pull request by RohitKandimalla**
…rrro-Severity option to filter out warnings

## CQL to ELM Translation Service PR

Jira Ticket: [MAT-8231](https://jira.cms.gov/browse/MAT-8231)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by adongare — APPROVED** [review]

File: src/main/java/gov/cms/mat/cql_elm_translation/service/EffectiveDataRequirementService.java [hunk 1]
```diff
@@ -30,7 +27,7 @@ public class EffectiveDataRequirementService {
   private RequestData createDefaultRequestData(String cql) {
     return RequestData.builder()
         .cqlData(cql)
-        .showWarnings(false)
+        .errorSeverity(CqlCompilerException.ErrorSeverity.Error)
```
Comment (adongare) [inline src/main/java/gov/cms/mat/cql_elm_translation/service/EffectiveDataRequirementService.java:30]: I think this is used for export. should this be configurable?

Comment (RohitKandimalla) [reply src/main/java/gov/cms/mat/cql_elm_translation/service/EffectiveDataRequirementService.java:30]: For now, We have future stories to handle what should be the error level. User will select from UI, then this selection will become dynamic
